website/package-lock.json

```diff
@@ -1,87 +1,121 @@
 {
   "name": "cortex-doc-hugo",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 3,
```

Comment:
I think you don't need to modify this file for this change. Can you double check?

Reply:
Ah yes, accidentally committed. I'll revert that thanks